src/router.ts

```diff
@@ -12,16 +12,14 @@ import * as utils from './requestHandlers/utils';
 
 export default (app: Express) => {
     app.get('/', help);
-
-    // Get by ID
-    app.get('/getHook', hook.getHook);
-    app.get('/getPanel', panel.getPanel);
-    app.get('/getPanelSetByID', panelSet.getPanelSetByID);
-    app.get('/getUserByID', user.getUserByID);
-    app.get('/getPanelBasedOnPanelSetAndIndex', panel.getPanelBasedOnPanelSetAndIndex);
-    app.get('/getPanelHooks', hook.getPanelHooks);
-    app.get('/getPanelsFromPanelSetID', panel.getPanelsFromPanelSetID); // documentation doesn't work for some reason
-    app.get('/getAllPanelSetsFromUser', panelSet.getAllPanelSetsFromUser); // documentation doesn't work for some reason
+    app.get('/hook/:id', hook.getHook);
+    app.get('/panel/:id', panel.getPanel);
+    app.get('panel/:id/hooks/', hook.getPanelHooks);
+    app.get('/panel_set/:id', panelSet.getPanelSetByID);
+    app.get('/panel_set/:id/panels/', panel.getPanelsFromPanelSetID); // documentation doesn't work for some reason
+    app.get('/panel_set/:panel_set_id/:index', panel.getPanelBasedOnPanelSetAndIndex);
```

Comment:
Should a `/panels` come after `:index` to remain consistent with the pattern of other requests?

Reply:
appended `/panel` instead as there should only be one